py-rse/errors.Rmd

```diff
@@ -27,19 +30,14 @@ Your Zipf's Law project should now include:
     └── ...
 ```
 
-In this chapter,
-we'll explore different types of errors,
-how they are communicated using code,
-and apply them to our Zipf's Law project.
-
-## What are exceptions? {#errors-what}
+## Exceptions {#errors-exceptions}
 
 Most modern programming languages use [exceptions][exception] for error handling.
 As the name suggests,
 an exception is a way to represent an exceptional or unusual occurrence
 that doesn't fit neatly into the program's expected operation.
-The code below shows a very simple example that tests different denominators and
-reports attempts to divide by zero:
+The code below shows a very simple example
+that uses exceptions to report attempts to divide by zero:
 
 ```python
```

Comment:
Following #342, this code block needs to be updated to use f-strings.

Reply:
-  Fixed those that can be converted (the error messages looked up in the dictionary can't be f-strings).
-  Altered `style.Rmd` to match.